pkg/cloudprivateipconfig/cloudprivateipconfig.go

```diff
@@ -0,0 +1,40 @@
+package cloudprivateipconfig
+
+import (
+	"errors"
+	"net"
+	"strings"
+)
+
+type IPFamily string
+
+const (
+	// IP Address Families
+	IPv4 IPFamily = "ipv4"
```

Comment:
nit: cloudprivateipconfig.go:14:2: exported const IPv6 should have comment (or a comment on this block) or be unexported